ci-operator/step-registry/p2p/mtv-spoke-to-hub-migration/p2p-mtv-spoke-to-hub-migration-commands.sh

```
@@ -0,0 +1,24 @@
+#
+# The p2p-mtv-spoke-to-hub-migration-ref.yaml declares P2P_MIGRATION_DIRECTION default
+# "spoke-to-hub" so the execute script picks up the direction even if the source call below
+# ever fails to resolve (e.g. flat-directory CI mount). In that case, update the ref.yaml
+# commands: field to the relative path resolved by the CI runner.
+#
+set -euo pipefail
+export P2P_MIGRATION_DIRECTION="spoke-to-hub"
+# The CI operator step-registry preserves the directory hierarchy when mounting scripts into
+# the step container. The execute script therefore resolves via the relative path below.
+# shellcheck source=../mtv-execute-hub-spoke-migration/p2p-mtv-execute-hub-spoke-migration-commands.sh
+source "$(dirname "$(realpath "${BASH_SOURCE[0]}")")/../mtv-execute-hub-spoke-migration/p2p-mtv-execute-hub-spoke-migration-commands.sh"
```

Shellcheck (0.11.0)

[info] Not following: ../mtv-execute-hub-spoke-migration/p2p-mtv-execute-hub-spoke-migration-commands.sh was not specified as input (see shellcheck -x).

(SC1091)